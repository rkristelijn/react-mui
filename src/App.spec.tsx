import React from 'react';
// eslint-disable-next-line import/no-extraneous-dependencies
import renderer from 'react-test-renderer';

import App from './App';

jest.mock('./ProTip', () => () => 'ProTip');

describe('App', () => {
  it('should initiate', () => {
    expect(App).toBeTruthy();
  });

  it('should match snapshop', () => {
    const component = renderer.create(<App />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
