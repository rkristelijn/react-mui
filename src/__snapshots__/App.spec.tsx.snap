// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshop 1`] = `
<div
  className="MuiContainer-root MuiContainer-maxWidthSm css-cuefkz-MuiContainer-root"
>
  <div
    className="MuiBox-root css-zitybv"
  >
    <h1
      className="MuiTypography-root MuiTypography-h4 MuiTypography-gutterBottom css-1vw6mcs-MuiTypography-root"
    >
      Material UI Create React App example in TypeScript
    </h1>
    ProTip
    <p
      className="MuiTypography-root MuiTypography-body2 MuiTypography-alignCenter css-1vl95ex-MuiTypography-root"
    >
      Copyright Â© 
      <a
        className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1cghntr-MuiTypography-root-MuiLink-root"
        href="https://mui.com/"
        onBlur={[Function]}
        onFocus={[Function]}
      >
        Your Website
      </a>
       
      2023
      .
    </p>
  </div>
</div>
`;
